src/xo-app/host/tab-network.js

```diff
@@ -189,63 +292,53 @@ class PifItem extends Component {
             className={pif.carrier ? 'text-success' : 'text-muted'}
           />
         </Tooltip>
-      </td>
-      <td className='text-xs-right'>
-        <ActionRowButton
-          disabled={pif.physical || pif.disallowUnplug || pif.management}
-          handler={deletePif}
-          handlerParam={pif}
-          icon='delete'
-          tooltip={_('deletePif')}
-        />
-      </td>
-    </tr>
+      </div>
+    },
+    name: _('pifStatusLabel')
   }
-}
+]
+const INDIVIDUAL_ACTIONS = [
+  {
+    handler: deletePif,
+    icon: 'delete',
+    label: _('deletePif')
+  }
+]
 
-export default ({
-  host,
-  networks,
-  pifs,
-  vifsByNetwork
-}) => <Container>
-  <Row>
-    <Col className='text-xs-right'>
-      <TabButton
-        btnStyle='primary'
-        handler={createNetwork}
-        handlerParam={host}
-        icon='add'
-        labelId='networkCreateButton'
-      />
-    </Col>
-  </Row>
-  <Row>
-    <Col>
-      {!isEmpty(pifs)
-        ? <span>
-          <table className='table'>
-            <thead className='thead-default'>
-              <tr>
-                <th>{_('pifDeviceLabel')}</th>
-                <th>{_('pifNetworkLabel')}</th>
-                <th>{_('pifVlanLabel')}</th>
-                <th>{_('pifAddressLabel')}</th>
-                <th>{_('pifModeLabel')}</th>
-                <th>{_('pifMacLabel')}</th>
-                <th>{_('pifMtuLabel')}</th>
-                <th>{_('defaultLockingMode')}</th>
-                <th>{_('pifStatusLabel')}</th>
-                <th className='text-xs-right'>{_('pifAction')}</th>
-              </tr>
-            </thead>
-            <tbody>
-              {map(pifs, pif => <PifItem key={pif.id} pif={pif} networks={networks} />)}
-            </tbody>
-          </table>
-        </span>
-        : <h4 className='text-xs-center'>{_('pifNoInterface')}</h4>
-      }
-    </Col>
-  </Row>
-</Container>
+export default class TabNetwork extends Component {
+  _groupedActions = [
+    {
+      label: 'deletePifs',
+      icon: 'delete',
+      handler: (ids) => deletePifs(filter(this.props.pifs, pif => includes(ids, pif.id)))
```

Comment:
Remove parenthesis around parameter when there's only one.

Reply:
`deletePifs` accepts PIF IDs as a parameter so you can pass `ids` directly. So this simplifies to:
```js
{
  // ...
  handler: deletePifs
}
```
and therefore, the grouped actions array doesn't need to be defined inside the class.